### DISCUSSION TITLE
Fetal error while installing openzeppelin

### DISCUSSION THREAD

**Discussion by Pushkarmondal** [Cyfrin/security-and-auditing-full-course-s23#133] [General]
I am getting error while installing openzeppelin. Please help 
this is the error = "forge install foundry-rs/forge-std --no-commit && forge install openzeppelin/openzeppelin-contracts@v3.4.0 --no-commit && forge install Brechtpd/base64 --no-commit
Installing forge-std in /Users/pushkarmondal/4-puppy-raffle-audit/lib/forge-std (url: Some("https://github.com/foundry-rs/forge-std"), tag: None)
Cloning into '/Users/pushkarmondal/4-puppy-raffle-audit/lib/forge-std'...
remote: Enumerating objects: 2168, done.
remote: Counting objects: 100% (2164/2164), done.
remote: Compressing objects: 100% (730/730), done.
remote: Total 2168 (delta 1438), reused 2060 (delta 1367), pack-reused 4
Receiving objects: 100% (2168/2168), 605.57 KiB | 78.00 KiB/s, done.
Resolving deltas: 100% (1438/1438), done.
Submodule 'lib/ds-test' (https://github.com/dapphub/ds-test) registered for path 'lib/forge-std/lib/ds-test'
Cloning into '/Users/pushkarmondal/4-puppy-raffle-audit/lib/forge-std/lib/ds-test'...
remote: Enumerating objects: 313, done.        
remote: Counting objects: 100% (171/171), done.        
remote: Compressing objects: 100% (79/79), done.        
remote: Total 313 (delta 91), reused 132 (delta 83), pack-reused 142        
Receiving objects: 100% (313/313), 71.35 KiB | 30.00 KiB/s, done.
Resolving deltas: 100% (130/130), done.
    Installed forge-std v1.7.6
Installing openzeppelin-contracts in /Users/pushkarmondal/4-puppy-raffle-audit/lib/openzeppelin-contracts (url: Some("https://github.com/openzeppelin/openzeppelin-contracts"), tag: Some("v3.4.0"))
Cloning into '/Users/pushkarmondal/4-puppy-raffle-audit/lib/openzeppelin-contracts'...
remote: Enumerating objects: 42568, done.
remote: Counting objects: 100% (870/870), done.
remote: Compressing objects: 100% (598/598), done.
error: RPC failed; curl 18 transfer closed with outstanding read data remaining
error: 183 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
fatal: clone of 'https://github.com/openzeppelin/openzeppelin-contracts' into submodule path '/Users/pushkarmondal/4-puppy-raffle-audit/lib/openzeppelin-contracts' failed
Error: "
![Uploading Screenshot 2024-02-25 at 15.00.01.png…]()

**Comment by EngrPips** [comment]
Hey @Pushkarmondal , installing openzeppelin contracts would always revert and return error messages whenever your internet connection is not very stable, i would say  check properly that your internet connection is strong and then try again.

**Comment by Pushkarmondal** [comment]
ok I will try
